easybuild/easyconfigs/a/Arb/Arb-2.16.0-iccifort-2018.3.222-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'Arb'
+version = '2.16.0'
+
+homepage = 'http://%(namelower)slib.org'
+
+description = """Arb is a C library for arbitrary-precision interval arithmetic.
+ It has full support for both real and complex numbers. The library is thread-safe,
+ portable, and extensively tested."""
+
+toolchain = {'name': 'iccifort', 'version': '2018.3.222-GCC-7.3.0-2.30'}
+
```

Comment:
Can't find the real reason for the failing test (not enough time), but add a
toolchainopts = {'noopt': True}
to Arb for the iccifort version and it will pass the tests.

Reply:
@SethosII Do you have any chance to test this build on a more recent CPU architecture and see if there are any other solutions aside from what Åke suggested?

Reply:
@vanzod I don't have the intel compiler, so I can't test the failing easyconfig myself.

Reply:
@SethosII Then please go ahead and apply the change that @akesandgren proposed.

Reply:
@vanzod @akesandgren I added the toolchain option.